### ISSUE TITLE
Sync Wiki issue?

### ISSUE THREAD

**Issue by rusefillc** [rusefi/rusefi_documentation#342]
![image](https://user-images.githubusercontent.com/48498823/210371243-4a823cf0-d5cf-42e5-9d18-012379b76568.png)

https://github.com/rusefi/rusefi/actions/runs/3830016361/jobs/6517357463

**Comment by rusefillc** [comment]
@chuckwagoncomputing do you understand root cause of issue?

**Comment by chuckwagoncomputing** [comment]
It seems this comes from https://github.com/rusefi/rusefi/pull/2355
Combined with breaking sync-wiki.yaml into two files.

**Comment by chuckwagoncomputing** [comment]
I'm at work now and can't test changes, but we should probably do the same thing to sync-wiki2.yaml and sync-wiki3.yaml that we are currently trying to do to sync-wiki.yaml in https://github.com/rusefi/rusefi/blob/master/.github/workflows/sync-wiki.yaml

**Comment by rusefillc** [comment]
sync-wiki.yaml reset/checkout magic makes no sense to me. While each line is documented I have no idea what's the big picture why.

**Comment by chuckwagoncomputing** [comment]
TL;DR bad magic github security policy that causes issues when someone commits a changed workflow file to rusefi/rusefi.wiki.git
As a workaround we undo any changes to workflow files.
